From 0fad0133b939f3b8b1163575ccdc41b1e7fb0b90 Mon Sep 17 00:00:00 2001
From: Sakari Ailus <sakari.ailus@linux.intel.com>
Date: Mon, 28 Sep 2015 13:00:41 +0300
Subject: [PATCH 04/19] media: v4l: Add 32-long vector YUV format define

Signed-off-by: Sakari Ailus <sakari.ailus@linux.intel.com>
---
 include/uapi/linux/videodev2.h | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/include/uapi/linux/videodev2.h b/include/uapi/linux/videodev2.h
index 1abe9d0..c1e4770 100644
--- a/include/uapi/linux/videodev2.h
+++ b/include/uapi/linux/videodev2.h
@@ -547,6 +547,10 @@ struct v4l2_pix_format {
 #define V4L2_PIX_FMT_YUV420M v4l2_fourcc('Y', 'M', '1', '2') /* 12  YUV420 planar */
 #define V4L2_PIX_FMT_YVU420M v4l2_fourcc('Y', 'M', '2', '1') /* 12  YVU420 planar */
 
+/* Vector YUV formats */
+#define V4L2_PIX_FMT_YYUV422_V32 v4l2_fourcc('y', 'V', '3', '2')
+#define V4L2_PIX_FMT_YYUV420_V32 v4l2_fourcc('y', '0', '3', '2')
+
 /* Bayer formats - see http://www.siliconimaging.com/RGB%20Bayer.htm */
 #define V4L2_PIX_FMT_SBGGR8  v4l2_fourcc('B', 'A', '8', '1') /*  8  BGBG.. GRGR.. */
 #define V4L2_PIX_FMT_SGBRG8  v4l2_fourcc('G', 'B', 'R', 'G') /*  8  GBGB.. RGRG.. */
-- 
2.8.1

